semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -64,23 +87,24 @@ message TextDocuments {
 }
 
 message TextDocument {
+  reserved 4;
   Schema schema = 1;
   string uri = 2;
   string text = 3;
-  string language = 4;
-  repeated SymbolOccurrence occurrences = 5;
-  repeated SymbolInformation symbols = 6;
+  Language language = 9;
+  repeated SymbolInformation symbols = 5;
```

Comment:
Should remain unchanged

Reply:
This is just synchronizing the spec with the protobuf file.